src/utils/image.ts

```diff
@@ -0,0 +1,8 @@
+import { Asset } from '@/api/queries'
+
+export const getImageUrlFromAsset = (image?: Asset | null) => {
```

Comment:
This should:
a) also handle `AssetStorage`, just like the code in `videoJsPlayer.ts`
b) be called `getUrlFromAsset` - it will handle videos as well
c) probably be moved into `src/utils/asset.ts`